internal/work/bundle.go

```diff
@@ -146,12 +146,12 @@ func NewPackageBundleBuilder(pkg Package, segmentRootLookup map[crypto.Hash]cryp
 	for itemIndex, item := range pkg.WorkItems {
 		for extrIndex, extr := range item.Extrinsics {
 			if int(offset+extr.Length) > len(extrinsics) {
-				return nil, fmt.Errorf("extrinsic data provided too small, expected length %d for item %d extrinsic %d", extr.Length, itemIndex, extrIndex)
+				return nil, fmt.Errorf("invalid extrinsic reference item=%d extr=%d hash=%x len=%d", itemIndex, extrIndex, extr.Hash, extr.Length)
```

Comment:
Did you mean to update this one too? Shouldn't this be something about the data being too small?